app/globals.css

```
@@ -117,6 +117,7 @@
 
   html {
     @apply bg-background font-sans text-foreground;
+    scroll-behavior: smooth;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)